mobile/src/modules/auth/screens/ImportAccountScreen.tsx

```diff
@@ -61,6 +62,7 @@ class ImportAccount extends React.PureComponent<Props, State> {
     try {
       const account = await this.props.dispatch(createOfflineAccount(address));
       this.props.dispatch(addAccount(account));
+      this.props.dispatch(hydrateAccount(account));
```

Comment:
todo: will rename this to `synchronizeAccount` to be consistent with the angular app